.gitignore

```diff
@@ -36,6 +37,7 @@ Temporary Items
 logs
 *.log
 npm-debug.log*
+package-lock.json
```

Comment:
It's recommended that you commit your lock files. See [here](https://docs.npmjs.com/cli/v6/configuring-npm/package-locks/#:~:text=It%20is%20highly%20recommended%20you,that%20you%20were%20developing%20on.)

Reply:
Updated